app/_locales/en/messages.json

```diff
@@ -2079,10 +2097,6 @@
   "userName": {
     "message": "Username"
   },
-  "verifyThisTokenOn": {
-    "message": "Verify this token on $1",
```

Comment:
This is fairly similar to the message for `swapsConfirmTokenAddressOnEtherscan` :thinking:

Is this close enough to keep around for the other locales, rather than deleting those translations?

Reply:
Do you mean use `swapsConfirmTokenAddressOnEtherscan` for english and `verifyThisTokenOn` for other locales? Or keep `verifyThisTokenOn`, and just not use `swapsConfirmTokenAddressOnEtherscan`?

Reply:
Design and product happy to keep original. Readd in bb2e461be

Reply:
To be clear, what I meant to suggest was that we change it for English but keep the original for translations, so they wouldn't lose a translation for this message. But this works too!